script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -818,25 +821,25 @@ If the lines are blood vessels,
 -- Page 486, Offset 17778.
 -- 「『モノの死に易い線』と『その死』……なんでしょうね、それ。
 // Identical phrase present in 04_00_ARC04_6_2D
-"'The seams of life' and 'death' itself... I wonder what it all means.
+"'Lines where things are easily killed' and 'death' itself... I wonder what it all means.
 }
 [sha:26c0af2eb669fdcf7d6cf14f2c6b2bfc76d5274c]{
 -- Page 486, Offset 17779.
 -- ……志貴はその気になれば鉱物の死を視ることだって可能だと思う。けどそのために脳が生物の範疇から鉱物の範疇に回線を開いて、鉱物の死を識ろうとしてしまう。
-...If you really wanted to, I think it'd be possible for you to see the death of inorganic matter.
- But in order to do that, your brain would have to open a circuit connecting the category of living things
- to the category of inanimate objects to allow you to comprehend their death.
+If you really wanted to, I think it'd be possible for you to see the death of inorganic matter.
+ But in order to do that, your brain's transmission lines have to go from being receptive to the living category,
```

Comment:
with all this line, EMR, etc. nonsense, I wonder if "category" wouldn't be well localized as "frequency" because "category" does sound pretty dumb, but it's also quite a _significant_ change.

Reply:
I kinda like that suggestion, I think 'living category' sounds stupid too.

Reply:
Frequency would definitely be the clearer loc here if it isn't Wrong™

Reply:
It in line with the metaphor, I think its fine.